.github/workflows/tests.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     -
       name: Install dependencies
       run: |
+        MAVLINK_ROUTER_COMMIT="94c4e3c6a9ff7c517b20b417e232caf52f12a6c6"
```

Comment:
Add a comment explaining where this magic hash is necessary and where we should look to find if is not in sync anymore.